src/config/mod.rs

```diff
@@ -28,6 +28,14 @@ pub fn webhook_token() -> String {
     env::var("WEBHOOK_TOKEN").expect("WEBHOOK_TOKEN missing in env")
 }
 
+pub fn transaction_service_auth_token() -> String {
+    let token = env::var("TRANSACTION_SERVICE_AUTH_TOKEN").unwrap_or_else(|_| {
+        log::warn!("TRANSACTION_SERVICE_AUTH_TOKEN missing in env");
+        String::new()
+    });
+    format!("Token {}", token)
```

Comment:
I think we can move this to the `add_header` just to make it more explicit that the `Token` is actually only required for the header and not part of the token itself wdyt?

Reply:
My rationale here was that `Authorization: Token token` gets handled as key value `("Authorization", "Token token")` therefore, it is always expected like that. Looking at this [link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes) I thought that maybe in some cases (in the  future) we would have `Bearer` or `Basic` and it would always be the same.
I am not sure though, I am totally open to suggestions. I would maybe move this `format!` to the `handler.rs` file 🤔 . Wdyt?

Reply:
> My rationale here was that `Authorization: Token token` gets handled as key value `("Authorization", "Token token")` therefore, it is always expected like that. Looking at this [link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes) I thought that maybe in some cases (in the future) we would have `Bearer` or `Basic` and it would always be the same.
> I am not sure though, I am totally open to suggestions. I would maybe move this `format!` to the `handler.rs` file 🤔 . Wdyt?

I was actually just thinking that the token by itself is not directly tied to the header in the `safe-transaction-service` even though it is the only use case so far. I was wondering if the authorisation token was used somewhere else besides the header then the CGW would already have that value in `TRANSACTION_SERVICE_AUTH_TOKEN` and we wouldn't need to strip the `Token ` part away.

This is not an issue. Was just something that I was considering and we can do this at a later point if needed.